### ISSUE TITLE
no more pd externals?

### ISSUE THREAD

**Issue by porres** [jamiebullock/LibXtract#101]
Hi, I can't see Pd externals anymore, and can't find them anywhere for download :/

**Comment by jamiebullock** [comment]
Yes, unfortunately that's true, see #74. If there is demand for a Pd external (and there does seem to be some) then I can look at making time to reinstate it

**Comment by porres** [comment]
Please do so, I could help you with that :) - build them and upload them to deken and all
